README.rst

```diff
@@ -638,6 +719,36 @@ change testingFacadeReal to True. It needs the usual OpenStack environment
 variables (*OS_USERNAME*, *OS_PASSWORD*, *OS_TENANT_NAME*, *OS_REGION_NAME*,
 *OS_AUTH_URL*)
 
+Contributing new tests
+______________________
+
+It is possible to contribute new tests defining a scenario in *tests/resources*
+For a scenario 'new_scenario', the following folders must be created:
+
+* new_scenario: there are files for each region with the backup of the metadata
+                BEFORE invoking the synchronisation. These files can be generated
+                with *sync.py --make-backup*
+* new_scenario.result: there are files for each region with teh backup of the
+                       metadata AFTER invoking the synchronisation
+* new_scenario_pre: there are files with the status of each region BEFORE invoking
+                    the synchronisation. These files can be generated with the
+                    output of *sync.py --show-status*
+* new_scenario_post: there are files with the status of each region AFTER invoking
+                     the synchronisation. These files can be generated with the
```

Comment:
You are right. Added missing line and also a paragraph about the optional config file.